pkg/connector/iam_user.go

```diff
@@ -111,6 +125,63 @@ func iamUserProfile(ctx context.Context, user iamTypes.User) map[string]interfac
 	return profile
 }
 
+func getLastLogin(ctx context.Context, client *iam.Client, user iamTypes.User) *time.Time {
+	logger := ctxzap.Extract(ctx).With(
+		zap.String("user_id", *user.UserId),
+	)
+
+	res, err := client.ListAccessKeys(ctx, &iam.ListAccessKeysInput{UserName: user.UserName})
+	if err != nil {
+		logger.Error("Error listing access keys", zap.Error(err))
+		return nil
+	}
+
+	accessKeyIDs := []string{}
+	for _, accessKey := range res.AccessKeyMetadata {
+		accessKeyIDs = append(accessKeyIDs, awsSdk.ToString(accessKey.AccessKeyId))
+	}
+
+	accessKeyLastUsedDates := make([]time.Time, 0, len(accessKeyIDs))
+	for _, accessKeyId := range accessKeyIDs {
```

Comment:
This seems like it'd slow down syncing quite a bit for accounts with lots of users (or lots of access keys). 

If it becomes a problem, we can add a config option to skip this. But until then I don't think it's worth adding yet another config option.